package.json

```diff
@@ -36,22 +35,20 @@
     "lib/"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
-    "@launchpadlab/babel-preset": "^2.1.0",
-    "@launchpadlab/eslint-config": "^2.4.1",
-    "axios": "^0.18.0",
```

Comment:
We're mocking this package entirely, so we don't actually need to install it locally for anything